lib/routemaster/drain/expiring_cache.rb

```diff
@@ -0,0 +1,38 @@
+require 'routemaster/middleware/root_post_only'
+require 'routemaster/middleware/authenticate'
+require 'routemaster/middleware/parse'
+require 'routemaster/middleware/expire_cache'
+require 'routemaster/middleware/payload_filter'
+require 'routemaster/middleware/filter'
+require 'routemaster/drain/terminator'
+require 'rack/builder'
+require 'delegate'
+
+module Routemaster
+  module Drain
+    # Rack application which authenticates, parses, filters duplicates in this request
+    # and invalidates the cache for all updated or new items.
+    #
+    # See the various corresponding middleware for details on operation:
+    # {Middleware::RootPostOnly}, {Middleware::Authenticate},
+    # {Middleware::Parse}, {Middleware::ExpireCache} and {Terminator}.
+    class ExpiringCache
```

Comment:
Naming: shouldn't this be `CacheExpiring`, as in "a middleware that expires the cache" (or even better `CacheBusting`)?

This reads as "a middleware that is or has an expiring cache" and seems a little confusing (to me!!)